llvm-hs/src/LLVM/Internal/Operand.hs

```diff
@@ -633,7 +633,9 @@ instance EncodeM EncodeAST A.MDNode (Ptr FFI.MDNode) where
     pure (FFI.upCast (ptr :: Ptr FFI.DINode))
   encodeM (A.DILocation {..}) = do
     Context c <- gets encodeStateContext
-    scopePtr <- encodeM locationScope
+    -- TODO Implement encoding of DILocalScope
```

Comment:
ah my bad, I didn't commit my (partial) implementation of encoding for DILocalScope